src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,19 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder binary = new StringBuilder();
+        int remainder;
```

Comment:
let's solve without this variable `remainder`

Reply:
done